apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -293,11 +294,20 @@ class InstructionsCSF extends React.Component {
     return this.instructions ? getOuterHeight(this.instructions, true) : 0;
   }
 
+  setInstructionsRef(instructions) {
+    this.instructions =
+      instructions && instructions.getWrappedInstance().instructions;
+
+    // TopInstructions, our parent, needs a ref to the body of the instructions
+    // to determine its height.
+    this.props?.setInstructionsRef(instructions);
```

Comment:
not concerned about this, but it seems odd -- can `this.props` ever be nullish?

Reply:
this.props is immutable, FYI. I don't see why it would be nullish though.